### PR TITLE
115 schedule page

### PR DESCRIPTION
**Pull request by sophiedeziel**

### CODE REVIEW

File: app/views/pages/schedule.html.erb [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      <%= l(Time.new(2016, 9, 23, 9, 15), :format => :time) %> - <%= l(Time.new(2016, 9, 23, 10, 30), :format => :time) %>
+    </div>
+    <div class="col-md-7">
+      <h4><%= t('.workshop.class.title') %></h4>
+      <%= t('.workshop.class.body') %>
+
+      <hr>
+      <div class="row">
+        <div align="center" class="col-md-4">
+          <a href=http://docs.railsbridge.org/ruby/><%= t('.workshop.class.lvl1') %></a>
```
Comment (steakunderscore) [inline app/views/pages/schedule.html.erb]: Do we want to be using http://docs.railsbridge-montreal.com/docs/?

Comment (sophiedeziel) [inline app/views/pages/schedule.html.erb]: Yes!!! I forgot to change the url!

Thank you Henry!

**Comment by steakunderscore** [comment]
Sophie if you are happy with my fixes you should merge! :shipit:

**Comment by sophiedeziel** [comment]
Thank you @steakunderscore !! That was exactly what needed to be done!

**Pull request merged**